core/modules/layout/tests/layout.test

```diff
@@ -1211,8 +1219,15 @@ class LayoutInterfaceTest extends BackdropWebTestCase {
   /**
    * Check that all the blocks in $layout match those in $copied_from_layout.
    */
-  protected function assertBlocksMatch($layout, $copied_from_layout) {
+  protected function assertBlocksMatch($layout, $copied_from_layout, $orphaned_uuids = array()) {
```

Comment:
If we decide to change the function parameters, we'll need to update the code comment block to include the new `$orphaned_uuids`

Reply:
I had looked at that. Right now the function doesn't list any parameters in the docblock.